skills_processor/fetch_raw_data.py

```diff
@@ -1,10 +1,12 @@
 # native packs
 import requests
 import json
+from typing import List
+
 # installed packs
 import pandas as pd
 from pandas import json_normalize
-
+# TODO this script is not used
```

Comment:
This script was used to fetch the EMSI skill database.  
It will be used later to update the database as it is updated frequently